### PR TITLE
Show message to exit video fullscreen mode

### PR DESCRIPTION
**Pull request by Ljung323**
Show how to exit video fullscreen mode when mouse hovers on the video view (https://github.com/mavlink/qgroundcontrol/issues/9563#issuecomment-808545622)

https://user-images.githubusercontent.com/82586164/115717387-1912c280-a3b5-11eb-97cc-c534cf12b950.mov

### CODE REVIEW

File: src/FlightDisplay/FlyViewVideo.qml [hunk 1]
```diff
@@ -63,10 +64,21 @@ Item {
...
+    QGCLabel {
+        text: qsTr("Double-click to exit full screen")
+        font.pointSize: ScreenTools.largeFontPointSize
+        visible: QGroundControl.videoManager.fullScreen && flyViewVideoMouseArea.isHovered
+        anchors.centerIn: parent
+    }
+
     MouseArea {
+        id: flyViewVideoMouseArea
+        property bool isHovered: false
```
Comment (DonLakeFlyer) [inline src/FlightDisplay/FlyViewVideo.qml:76]: Why is this property needed? Why can't `flyViewVideoMouseArea.hovered` be used directly?

Comment (Ljung323) [reply src/FlightDisplay/FlyViewVideo.qml:76]: @DonLakeFlyer 
I tried to use the pre-defined `hovered`, but it is not defined for MouseArea. 
([Button](https://doc.qt.io/qt-5/qml-qtquick-controls2-control.html#hovered-prop) has `hovered` property, but it seems [MouseArea](https://doc.qt.io/qt-5/qml-qtquick-mousearea-members.html) does not have it)

Comment (Ljung323) [reply src/FlightDisplay/FlyViewVideo.qml:76]: Also, renaming `isHovered` to `hovered` does not work. (other names than `hovered` works)
I don't figure out the reason, but when `property bool hovered` is defined, `onHoveredChanged` is not triggered.

Comment (DonLakeFlyer) [reply src/FlightDisplay/FlyViewVideo.qml:76]: > I tried to use the pre-defined `hovered`, but it is not defined for MouseArea.

That's certainly odd. Looks like it is 'containsMouse` from MouseArea which tells you what you need. You should be able to use that as the reference for showing/hiding the help test.

Comment (Ljung323) [reply src/FlightDisplay/FlyViewVideo.qml]: @DonLakeFlyer 
Thank you for the feedback, `containsMouse` worked perfectly. a9c0d52
On the first trial for `containsMouse`, I did not enable `hoverEnabled` which triggers `containsMouse`.

https://user-images.githubusercontent.com/82586164/119072360-4d19eb80-ba26-11eb-92a3-bba7173afee2.mov

Comment (DonLakeFlyer) [reply src/FlightDisplay/FlyViewVideo.qml]: Great. Much simpler now. Thanks.

**Pull request merged**